### PR TITLE
add convolution computation

### PR DESCRIPTION
**Pull request by amandazhang**
For the data we need to make analysis on, we have cond.txt file under 'Model--Modelx--onset--taskxxx_runxxx--condxxx.txt'. We need to use cond.txt file to get the convolution and save as conv.txt.
This commitment only lay out the basic idea on getting convolution, it's not written as a general function to have input cond.txt and output as conv.txt.
Also on the numbers(those constants) we use in the .py file, anyone know the meaning of them and why instructor set those values?

### CODE REVIEW

**Comment by Jay4869** [comment]
let's focus `convolution computation` in my branch. Is that okay for u? If yes, please close this

**Comment by amandazhang** [comment]
@Jay4869  Answer my question first! Then I'll consider about closing it. :)

**Comment by Jay4869** [comment]
what numbers (constants) are you confused? You mean task#, run#, and cond#?

File: convolution.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import scipy.stats
+from scipy.stats import gamma
+from stimuli import events2neural
+
+def hrf(times):
```
Comment (amandazhang) [inline convolution.py]: @Jay4869 6, 12, 0.35, ect in the function hrf

File: convolution.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    peak_values = gamma.pdf(times,6)
+    undershoot_values = gamma.pdf(times, 12)
+    values = peak_values-0.35*undershoot_values
+    
+    return values/np.max(values)*0.6
```
Comment (amandazhang) [inline convolution.py]: @Jay4869 6,12,0.35, 0.6 in the hrf function

Comment (Jay4869) [inline convolution.py]: Based on lecture, I remembered it is just a way to combine different gamma distributions  in order to form a distribution has peak and lowest value. We can change these parameter to see how is perform, but prof did go through math theory behind

**Pull request closed without merging**